ros_gz_example_description/models/diff_drive/model.sdf

```diff
@@ -164,7 +164,12 @@
         <sensor name='gpu_lidar' type='gpu_lidar'>
           <pose>0 0 0 0 0 0</pose>
           <topic>scan</topic>
+          <!-- 
+             gzwarn << "The `ignition_frame_id` tag is deprecated. "
+               << "Please use `gz_frame_id` instead." << std::endl;
           <ignition_frame_id>diff_drive/lidar_link</ignition_frame_id>
+          -->
```

Comment:
```suggestion
```

Delete this comment, the old stuff is deprecated and not needed.

Reply:
Thanks for the feedback, was unsure how to do it.

Reply:
With this change, does the demo run for you in harmonic?

Reply:
Yes I can start the simulation of diff drive. Though I have no use case to check the correct frame id settings.

Reply:
What happens when you run these commands? Please include the logs of the simulator from bootup if you can. For me, I have other problems in the logs, and the Gazebo model does not move.
https://github.com/gazebosim/ros_gz_project_template/blob/e70509303e59e81183d1de8d998ff680dc1759a7/ros_gz_example_gazebo/worlds/diff_drive.sdf#L3-L8

Reply:
I recreated the workspace with my fork as repository. 
I build the workspace and run the launch file
I Could run the the above command gz topic -t "/model/diff_drive/cmd_vel" -m gz.msgs.Twist -p "linear: {x: 1.0}, angular: {z: -0.1}"  successfully. 
The robot is moving. 
I added a cube in the simulation to see, if the lidar is working (check function of the frame id)
I configured the Laser Scan in Rviz2
I Can see the resulting laser scan 

I think the functionality is working. Do i miss something ?
I am running humble and gz sim 8.3.0

![diffdrive-test](https://github.com/gazebosim/ros_gz_project_template/assets/13600164/afbdc1fb-ed76-4503-a07c-f3cbefdfe29a)

Reply:
Great information, that's different behavior than me. Can you attach the terminal logs from running the `gz sim` command so I can compare?

Reply:
Are you still interested in the logs? Because the or was merged.

Reply:
Yes, I am.

Reply:
This is my console log. I hope this is what you wanted 

ros2 launch ros_gz_example_bringup diff_drive.launch.py
[INFO] [launch]: All log files can be found below /home/marvin/.ros/log/2024-04-26-19-35-34-223355-marvin-Aspire-13252
[INFO] [launch]: Default logging verbosity is set to INFO
[INFO] [ruby $(which gz) sim-1]: process started with pid [13253]
[INFO] [parameter_bridge-2]: process started with pid [13255]
[INFO] [robot_state_publisher-3]: process started with pid [13258]
[INFO] [rviz2-4]: process started with pid [13261]
[robot_state_publisher-3] Warning [Utils.cc:132] [/sdf/model[@name="diff_drive"]/link[@name="lidar_link"]/sensor[@name="gpu_lidar"]/gz_frame_id:<data-string>:L167]: XML Element[gz_frame_id], child of element[sensor], not defined in SDF. Copying[gz_frame_id] as children of [sensor].
[robot_state_publisher-3] [WARN] [1714152935.493556544] [sdformat_urdf]: SDFormat link [lidar_link] has a <sensor>, but URDF does not support this
[robot_state_publisher-3] [WARN] [1714152935.505687487] [kdl_parser]: The root link chassis has an inertia specified in the URDF, but KDL does not support a root link with an inertia.  As a workaround, you can add an extra dummy link to your URDF.
[robot_state_publisher-3] [WARN] [1714152935.506851398] [kdl_parser]: Converting unknown joint type of joint 'caster_wheel' into a fixed joint
[robot_state_publisher-3] [INFO] [1714152935.506900424] [robot_state_publisher]: got segment caster
[robot_state_publisher-3] [INFO] [1714152935.506997850] [robot_state_publisher]: got segment chassis
[robot_state_publisher-3] [INFO] [1714152935.507014003] [robot_state_publisher]: got segment left_wheel
[robot_state_publisher-3] [INFO] [1714152935.507026615] [robot_state_publisher]: got segment lidar_link
[robot_state_publisher-3] [INFO] [1714152935.507038567] [robot_state_publisher]: got segment right_wheel
[robot_state_publisher-3] [INFO] [1714152935.507070049] [robot_state_publisher]: Floating joint. Not adding segment from chassis to caster.
[ruby $(which gz) sim-1] Warning [Utils.cc:132] [/sdf/model[@name="diff_drive"]/link[@name="lidar_link"]/sensor[@name="gpu_lidar"]/gz_frame_id:/home/marvin/template_ws/install/ros_gz_example_description/share/ros_gz_example_description/models/diff_drive/model.sdf:L167]: XML Element[gz_frame_id], child of element[sensor], not defined in SDF. Copying[gz_frame_id] as children of [sensor].
[rviz2-4] [INFO] [1714152936.715373710] [rviz2]: Stereo is NOT SUPPORTED
[rviz2-4] [INFO] [1714152936.715481869] [rviz2]: OpenGl version: 4.6 (GLSL 4.6)
[rviz2-4] [INFO] [1714152936.825188219] [rviz2]: Stereo is NOT SUPPORTED
[parameter_bridge-2] [INFO] [1714152937.136947365] [ros_gz_bridge]: Creating ROS->GZ Bridge: [/diff_drive/cmd_vel (geometry_msgs/msg/Twist) -> /model/diff_drive/cmd_vel (gz.msgs.Twist)] (Lazy 0)
[parameter_bridge-2] [INFO] [1714152937.162903569] [ros_gz_bridge]: Creating GZ->ROS Bridge: [/clock (gz.msgs.Clock) -> /clock (rosgraph_msgs/msg/Clock)] (Lazy 0)
[parameter_bridge-2] [INFO] [1714152937.166674219] [ros_gz_bridge]: Creating GZ->ROS Bridge: [/model/diff_drive/odometry (gz.msgs.Odometry) -> /diff_drive/odometry (nav_msgs/msg/Odometry)] (Lazy 0)
[parameter_bridge-2] [INFO] [1714152937.170679319] [ros_gz_bridge]: Creating GZ->ROS Bridge: [/scan (gz.msgs.LaserScan) -> /diff_drive/scan (sensor_msgs/msg/LaserScan)] (Lazy 0)
[parameter_bridge-2] [INFO] [1714152937.184142887] [ros_gz_bridge]: Creating GZ->ROS Bridge: [/world/demo/model/diff_drive/joint_state (gz.msgs.Model) -> /joint_states (sensor_msgs/msg/JointState)] (Lazy 0)
[parameter_bridge-2] [INFO] [1714152937.186386294] [ros_gz_bridge]: Creating GZ->ROS Bridge: [/model/diff_drive/pose (gz.msgs.Pose_V) -> /tf (tf2_msgs/msg/TFMessage)] (Lazy 0)
[parameter_bridge-2] [INFO] [1714152937.192660021] [ros_gz_bridge]: Creating GZ->ROS Bridge: [/model/diff_drive/pose_static (gz.msgs.Pose_V) -> /tf_static (tf2_msgs/msg/TFMessage)] (Lazy 0)
[rviz2-4] Warning [Utils.cc:132] [/sdf/model[@name="diff_drive"]/link[@name="lidar_link"]/sensor[@name="gpu_lidar"]/gz_frame_id:<data-string>:L167]: XML Element[gz_frame_id], child of element[sensor], not defined in SDF. Copying[gz_frame_id] as children of [sensor].
[rviz2-4] [WARN] [1714152937.246291946] [sdformat_urdf]: SDFormat link [lidar_link] has a <sensor>, but URDF does not support this
[ruby $(which gz) sim-1] libEGL warning: egl: failed to create dri2 screen
[ruby $(which gz) sim-1] libEGL warning: egl: failed to create dri2 screen
[ruby $(which gz) sim-1] libEGL warning: egl: failed to create dri2 screen
[ruby $(which gz) sim-1] libEGL warning: egl: failed to create dri2 screen